### ISSUE TITLE
Flutter App is not working for Dart 3

### ISSUE THREAD

**Issue by maximiliankipping** [DevStack06/Whatsapp-Clone-Flutter#7]
Hello, 
I wanted to follow your programm sessions to programm my own Chat App. But your instructions are most of the time not working because it is not compatible with Dart 3. Could you please update the Code? Furthermore some Package like the one for Emojis are not Dart 3 compatible

**Comment by rranjithkumar100** [comment]
yes, same issue happens. Planning to buy the full project, but it seems not maintainable project.
